src/BrowserKitDriver.php

```diff
@@ -236,8 +236,14 @@ public function setCookie($name, $value = null)
             return;
         }
 
+        try {
+            $domain = parse_url($this->getCurrentUrl(), PHP_URL_HOST) ?: '';
+        } catch (DriverException $exception) {
```

Comment:
@aik099 this will happen if you have not yet visited a page.

And if you write code to detect this here, you will duplicate the code of `$this->getCurrentUrl()` (as you will have to get the request to know whether it is null or no)